### PR TITLE
Add mfa support

### PR DESCRIPTION
**Pull request by renkelvin**

### CODE REVIEW

**Comment by bojeil-google** [comment]
Hey @morganchen12, we will initially add documentation/snippets in [GCIP docs](https://cloud.google.com/identity-platform/). We will follow up and add to Firebase docs once this is available in the Firebase Console. One thing to really consider is to support this in FirebaseUI (Android, iOS and web). I think with sufficient demand, we can justify that.

**Comment by morganchen12** [comment]
Makes sense, thanks for clarifying.

**Comment by renkelvin** [comment]
@samtstern @morganchen12 Please take a look at the latest commit, which adds a new sign in entry "Email with MFA" and keeps the regular "Email" unchanged.

File: authentication/AuthenticationExampleSwift/MainViewController.swift [hunk 1]
```diff
@@ -28,6 +28,8 @@ import GoogleSignIn
 import FBSDKCoreKit
 import FBSDKLoginKit
 
+var isMFAEnabled = false
```
Comment (morganchen12) [inline authentication/AuthenticationExampleSwift/MainViewController.swift]: nit: make this private

**Pull request merged**